README.md

```diff
@@ -448,3 +448,17 @@ template.hbs:
 
 {{load-more-button action='infinityLoad' infinityModel=model}}
 ```
+
+### Delay start of infinite loading until user has indicated they would like to load more
+
+Infinite loading need not start automatically, it may be kickstarted from a user action. The infinity-loader component can be popped in and out, eg: each time a new search is entered, the component may be removed, and then put back in place at the user’s request.
+
```

Comment:
Could you explain the second sentence?  I'm thinking the first sentence only good enough for this example.  What do you think?

Reply:
@snewcomer Oh maybe so! –– I wanted to make it explicit in case it wasn't obvious that this will work repeatedly – much to my delight! – as in our use case where we have a search page where the user can adjust the search params to their heart’s content, and each time they do that, we toggle the boolean back to false, and it just works.

Reply:
Gotcha.  That seems cool!  Yeah I think the first sentence is what will apply to everybody so if you don't mind ✂️ the second sentence, that would be sweet.

Reply:
@snewcomer Ok will do!